packages/frama-c/frama-c.24.0/opam

```diff
@@ -128,7 +128,7 @@ depopts: [
   # Coq: because .vo would would not be loadable by another version of Coq
   # libraries: because we use dynamic linking
   "apron"
-  "coq"
+  "coq" { < "8.16.1" }
```

Comment:
```suggestion
  "coq" { < "8.16" }
```
We can see in CI runs for https://github.com/ocaml/opam-repository/pull/22564 and https://github.com/ocaml/opam-repository/pull/22066 that `frama-c.24.0` fails as a reverse dependency. However, this could also be due to a general problem with the package (and not with Coq version specifically) .

Reply:
precisely, so I'm not sure why the author added an unrelated constraint?